content_handler/BUILD.gn

```diff
@@ -9,6 +9,12 @@ import("//build/vulkan/config.gni")
 template("flutter_content_handler") {
   invoker_output_name = invoker.output_name
   extra_deps = invoker.extra_deps
+  if (invoker.flx) {
+    runtime_holder = [ "runtime_holder.cc" ]
+  } else {
+    # Replace runtime_holder.cc with runtime_holder_no_flx.cc when flx is removed.
+    runtime_holder = [ "runtime_holder_no_flx.cc" ]
```

Comment:
We shouldn't need to fork runtime_holder.cc.  Instead, we should be able to add a few if statements to it to handle both cases.